modules/volume/data/regionmap.csv

```diff
@@ -0,0 +1,29 @@
+﻿pre;new
```

Comment:
A suggestion is to refactor the code in the processor into a function, add a unit test for that function and move this file to the unit test. 
I would consider removing the workspace in that case.

Reply:
We also want a regression test with the processor, and the file is needed for that.

Reply:
I also want the processor to be tested.